### ISSUE TITLE
Creating multiple wood bucket

### ISSUE THREAD

**Issue by djoey123** [Jake-E/Myrtrees#8]
1. fill your inventory, so you have no space.
2. create multiple items with shift and click on outut
3. get 1 wooden bucket, everything else is deleted

Modpack: FTB Ultimate: Anniversary Edition 1.0.4

**Comment by Jake-E** [comment]
Is this whilst crafting?

**Comment by djoey123** [comment]
Yes, when you craft it, everything inb the crafting bench is removed, and you only get 1 wooden bucket
